package.json

```diff
@@ -211,6 +211,7 @@
     "tailwindcss": "^3.4.1",
     "ts-jest": "^29.1.2",
     "ts-node": "^10.9.1",
+    "tsc-alias": "^1.8.7",
     "typescript": "^5.3.3"
```

Comment:
Not sure I totally follow the current state of the build scripts and subplat prototypes, but I found this almost interesting:
```sh
git grep '"typescript":' -- package.json packages/**/package.json apps | cat

apps/payments/next/package.json:    "typescript": "5.0.4"
package.json:    "typescript": "^5.3.3"
packages/fxa-admin-panel/package.json:    "typescript": "^5.3.3",
packages/fxa-admin-server/package.json:    "typescript": "^5.3.3",
packages/fxa-auth-client/package.json:    "typescript": "^5.3.3",
packages/fxa-auth-server/package.json:    "typescript": "^5.3.3",
packages/fxa-content-server/package.json:    "typescript": "^5.3.3",
packages/fxa-event-broker/package.json:    "typescript": "^5.3.3"
packages/fxa-graphql-api/package.json:    "typescript": "^5.3.3"
packages/fxa-payments-server/package.json:    "typescript": "^5.3.3",
packages/fxa-react/package.json:    "typescript": "^5.3.3",
packages/fxa-settings/package.json:    "typescript": "^5.3.3",
packages/fxa-shared/package.json:    "typescript": "^5.3.3",
```

Where apps/payments/next/package.json is the outlier at TS 5.0.4 instead of ^5.3.3.

```sh
git grep -Phio '"typescript": ".*"' -- package.json packages/**/package.json apps/**/package.json | sort | uniq -c

   1 "typescript": "5.0.4"
  12 "typescript": "^5.3.3"
```

Reply:
Ah thank you. I manually updated payments-next to be the same typescript version as the others.

Regarding `tsc-alias`, we're using that to resolve the Typescript alias paths, primarily in scripts and JS files. 

We've started using in TS aliases in the `libs/` libraries.